### PR TITLE
Rename funct

### PR DESCRIPTION
**Pull request by outscale-toa**

### CODE REVIEW

File: tests/functions.sh [hunk 1]
```diff
@@ -64,14 +71,82 @@ function ssh_no_ping {
...
 
+function ssh_connection_test {
+    protocol=$1
+    id1=$2
+    id2=$3
+    port=$4
+
+    if [ "$protocol" == "udp" ]; then
+	ssh_run_background $id1 "nc -w 3 -lup $port > /tmp/test"
+	sleep 1
```
Comment (outscale-mgo) [inline tests/functions.sh]: what's the point of this sleep ?

File: tests/functions.sh [hunk 2]
```diff
@@ -64,14 +71,82 @@ function ssh_no_ping {
...
 }
 
+function ssh_connection_test {
+    protocol=$1
+    id1=$2
+    id2=$3
+    port=$4
+
+    if [ "$protocol" == "udp" ]; then
+	ssh_run_background $id1 "nc -w 3 -lup $port > /tmp/test"
```
Comment (outscale-mgo) [inline tests/functions.sh]: I thins 2 second is enouth

File: tests/functions.sh [hunk 3]
```diff
@@ -64,14 +71,82 @@ function ssh_no_ping {
...
+	sleep 1
+	ssh_run_background $id2 "echo 'this message is from vm $id2' | nc -u -w 3 42.0.0.$id1 $port"
```
Comment (outscale-mgo) [inline tests/functions.sh]: 1 seconde is enouth :)

File: tests/functions.sh [hunk 4]
```diff
@@ -64,14 +71,82 @@ function ssh_no_ping {
...
+	else
+	    echo -e "$protocol test $id2 --> $id1 FAIL"
+	    RETURN_CODE=1
+	fi
+	ssh_run $id1 "rm /tmp/test"
+    fi
+}
+
+function ssh_no_connection_test {
+    protocol=$1
```
Comment (outscale-mgo) [inline tests/functions.sh]: this function is way to similar to ssh_connection_test, can you refactor it ?

File: tests/functions.sh [hunk 5]
```diff
@@ -64,14 +71,82 @@ function ssh_no_ping {
...
+
 function qemu_start {
     id=$1
     echo "starting VM $id"
     SOCKET_PATH=/tmp/qemu-vhost-nic-$id
     IMG_PATH=$BUTTERFLY_BUILD_ROOT/vm.qcow
     MAC=52:54:00:12:34:0$id
 
-    CMD="sudo qemu-system-x86_64 -netdev user,id=network0,hostfwd=tcp::500${id}-:22 -device e1000,netdev=network0 -m 124M -enable-kvm -chardev socket,id=char0,path=$SOCKET_PATH -netdev type=vhost-user,id=mynet1,chardev=char0,vhostforce -device virtio-net-pci,mac=$MAC,netdev=mynet1 -object memory-backend-file,id=mem,size=124M,mem-path=/mnt/huge,share=on -numa node,memdev=mem -mem-prealloc -drive file=$IMG_PATH -snapshot -nographic"
+    CMD="sudo qemu-system-x86_64 -netdev user,id=network0,hostfwd=tcp::500${id}-:22 -device e1000,netdev=network0 -m 124M -enable-kvm -chardev socket,id=char0,path=$SOCKET_PATH -netdev type=vhost-user,id=mynet1,chardev=char0,vhostforce -device virtio-net-pci,csum=off,gso=off,mac=$MAC,netdev=mynet1 -object memory-backend-file,id=mem,size=124M,mem-path=/mnt/huge,share=on -numa node,memdev=mem -mem-prealloc -drive file=$IMG_PATH -snapshot -nographic"
```
Comment (outscale-mgo) [inline tests/functions.sh]: can you move this to another commit please

**Comment by outscale-mgo** [comment]
So you are breaking the tests in the 2nd commit and fixing it in the 3rd one...
can you move the add_nic change to the 3rd commit ?

**Comment by outscale-jju** [comment]
superseded by #92
